### ISSUE TITLE
Club import not working

### ISSUE THREAD

**Issue by jopreijnen** [kosciukiewicz/sbc-solver#5]
Club not imported. Tried multiple times but it does not import my club to the solver unfortunately.

**Comment by kosciukiewicz** [comment]
Does the sbc import work fine?

**Comment by jopreijnen** [comment]
fixed itself after a restart, think it collided with another extension i had on

**Comment by kosciukiewicz** [comment]
Glad to hear that 😄

**Comment by jopreijnen** [comment]
but a lot of the requirements are unsupported, the tool does not do foundations right? As it does not do the minimum gold.

**Comment by kosciukiewicz** [comment]
Yes, not all requirements are implemented - feel free to request the most desired one so I can implement it for You. 

I tested in on foundations with fc 25. I know some requirements are not implemented but still You can generate the solution (the no implemented requirements are not taken into account) - the solution might be satisfactory or easy to adjust by hand.

As for the minimum gold - that is not implemented at all - the solver does not have access to the market prices - what I can do relatively quickly is to take into account quick sell price but that's not accurate in most cases.
